test/e2e/framework/context_lifecycle_operations.go

```diff
@@ -21,7 +21,7 @@ type ContextCmdOps interface {
 	// GetContext helps to run `context get` command
 	GetContext(contextName string, opts ...E2EOption) (ContextInfo, error)
 	// ListContext helps to run `context list` command
-	ListContext(opts ...E2EOption) ([]*ContextListInfo, error)
+	ListContext(opts ...E2EOption) (contexts []*ContextListInfo, stdOutStr, stdErrStr string, err error)
```

Comment:
Just curious, why change this signature?  I don't see anywhere in this PR the use of `stdoutStr/stdErrStr`

Reply:
We do need to stdOut and stdErr some cases, thats why i am keep updating all our commands.